locksmith/src/operations/checkoutConfigOperations.ts

```diff
@@ -8,35 +13,101 @@ interface SaveCheckoutConfigArgs {
   config: any // TODO: TYPE
 }
 
-// Creates or updates a checkout config
+/**
+ * Extracts lock addresses and network information from the config
+ * @param config - The checkout configuration object
+ * @returns An array of objects containing lock address and network
+ */
+const extractLockInfo = (
+  config: any
+): { address: string; network: number }[] => {
+  return Object.entries(config.locks).map(
+    ([address, lockConfig]: [string, any]) => ({
+      address,
+      network: lockConfig.network,
```

Comment:
lockconfig.network may be missing. In that case, the paywallconfigwill have a "root" level network, like this:
```json
{
  locks: {
     '0x123' : {}
 },
 network: 123
}
```

Reply:
got it!